Dockerfile

```diff
@@ -1,10 +1,8 @@
-# This is the SHA of the submodule
-ARG BASE_TAG=5fd3380c
```

Comment:
No longer needed to add sha here since in the actions repo at v1.0.4 release. 
Locally you will need to ensure the `BASE_TAG` is setup to the most recent submodule's 8-digit sha and then build. Use the script at bin/submodule-update.sh or use stack_car that provides support: https://github.com/notch8/stack_car/blob/3f80d0e2b27c66a08d2563c62b8b151699798487/lib/stack_car/cli.rb#L430